redpanda/values.yaml

```diff
@@ -13,123 +13,217 @@
+    # provisioning.
+    # If undefined or empty (default), then no `storageClassName` spec is set,
+    # so the default provisioner will be chosen (gp2 on AWS, standard on
+    # GKE, AWS & OpenStack).
+    storageClass: ""
+    # Additional labels to apply to the created PersistentVolumeClaims.
+    labels: {}
+    # Additional annotations to apply to the created PersistentVolumeClaims.
+    annotations: {}
 
-  rpk:
-    # Enables memory locking.
-    enable_memory_locking: false
-
-    # Send usage stats back to vectorized
-    enable_usage_stats: true
-
-    # This should be set to true unless with have CPU affinity
-    # https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy
-    #
-    # Equivalent to passing the following to redpanda
-    # --idle-poll-time-us 0 --thread-affinity 0 --poll-aio 0
-    overprovisioned: true
-
-    # Increases the number of allowed asynchronous IO events
-    tune_aio_events: false
-
-    # Syncs NTP
-    tune_clocksource: false
-
-    # Installs a custom script to process coredumps and save them to the given directory.
-    tune_coredump: false
-
-    # Disables hyper-threading, sets the ACPI-cpufreq governor to 'performance'. Additionaly
-    # if system reboot is allowed: disables Intel P-States, disables Intel C-States,
-    # disables Turbo Boost
-    #
-    # This is not possible to set to true while running in a container.
-    tune_cpu: false
-
-    # Distributes IRQs across cores with the method deemed the most appropriate for the
-    # current device type (i.e. NVMe)
-    #
-    # This is not possible to set to true while running in a container.
-    tune_disk_irq: false
-
-  seastar:
-    default_log_level: info
+# Redpanda Keeper configuration
+rpk:
+  # Enables memory locking.
+  enable_memory_locking: false
+  # Send usage stats back to vectorized
+  enable_usage_stats: true
+  #
+  # This should be set to true unless with have CPU affinity
+  # https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy
+  # Equivalent to passing the following to redpanda
+  # --idle-poll-time-us 0 --thread-affinity 0 --poll-aio 0
+  overprovisioned: true
+  # Increases the number of allowed asynchronous IO events
+  tune_aio_events: false
+  # Syncs NTP
+  tune_clocksource: false
+  # Installs a custom script to process coredumps and save them to the given directory.
+  tune_coredump: false
+  #
+  # Disables hyper-threading, sets the ACPI-cpufreq governor to 'performance'.
+  # Also if system reboot is allowed it disables: Intel P-States, Intel C-States, and Turbo Boost
+  # Setting to true is not possible while running in a container.
+  tune_cpu: false
+  #
+  # Distributes IRQs across cores with the method deemed the most appropriate for the
+  # current device type (i.e. NVMe)
+  # Setting to true is not possible while running in a container.
+  tune_disk_irq: false
```

Comment:
We should split this config, one for tuners (tune_* params) and one for the rest (probably all within node configuration properties)

This might be unfamiliar for users that haven't tried rpk.

[questions] Do we run rpk tune during deployment/setup of the helm chart?

Reply:
> Can I enable global TLS and then disable it for a single listener with `enabled: false` here, right?

Yes.

> We should split this config, one for tuners (tune_* params) and one for the rest (probably all within node configuration properties)

Agreed, this is the change we discussed during our last call. I will create a PR that removes the `rpk` section and breaks up parameters into groups as you suggest (see https://github.com/redpanda-data/helm-charts/issues/107).

> Do we run rpk tune during deployment/setup of the helm chart?

No.